packages/realm-server/node-realm.ts

```diff
@@ -173,15 +176,18 @@ export class NodeAdapter implements RealmAdapter {
     };
   }
 
```

Comment:
`.write` method in any of the adapters is dealing with the filesystem write exclusively. There is no bulk method here.